### DISCUSSION TITLE
Any way to use TAB for forms while having Tour open?

### DISCUSSION THREAD

**Discussion by terragady** [gilbarbara/react-joyride#831] [Ideas]
I have a form fields to enter and when pressing tab it focuses on the tooltip of the tour rather then going down on the form, is there possibility to turn off keyboard shortcuts for Tour? I do not need any keyboard navigation for the tour.

**Comment by gilbarbara** [comment]
hey @terragady,
 
Not currently, the keyboard navigation is for accessibility.

**Comment by terragady** [comment]
hear you, but then it is not very accessible if you cant tab to your form to fill it :)

**Reply by gilbarbara** [reply]
If the user uses a screen reader and while the tour is running, every element is accessible with the tab key, he won't know what to do next. [What is accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)

Maybe you can pause the tour if the user wants to complete the form and restart it afterward?

**Reply by terragady** [reply]
well this is some kind of idea but the tour is on the page and form is in modal, it creates chaos at some point but yes I think I can manage to do something. 
with the TAB tour is the only accessible part at the moment if it running. Anyway thanks for helping, I will get over it somehow :)
